src/elements/content-sidebar/activity-feed/task-new/Task.js

```diff
@@ -105,9 +123,43 @@ class Task extends React.Component<Props, State> {
         );
     };
 
+    fetchTaskCollaborators = async () => {
+        const { id, api, task_links } = this.props;
+        const { entries } = task_links;
+        let fileId;
+
+        if (entries[0].target) {
+            fileId = entries[0].target.id;
+        }
+        this.setState({ isLoading: true });
+
+        return new Promise((resolve, reject) => {
```

Comment:
You have async above, but no await usage, should convert to use await

Reply:
API methods like `getTaskCollaborators` do not return a Promise, this wraps the callbacks to enable Promise chaining or async/await elsewhere in the component

Reply:
Removed async from signature.